src/state/selection/types.ts

```diff
@@ -30,13 +30,31 @@ export interface HighlightAgentAction {
     payload: string;
     type: string;
 }
-
-export interface VisibilitySelectionMap {
+/**
+ * AgentRenderingCheckboxMap represents the state of the visibility checkboxes
+ * and the highlight checkboxes
+ * In the case of agents with no displaystates/children:
```

Comment:
they are synonyms: `displayState` is the trajectory data way of thinking about them and `children` is the checkbox way of thinking of them since they are nested under the parent checkbox